netbox/dcim/svg/racks.py

```diff
@@ -48,13 +48,15 @@ def get_device_description(device):
 
     Name: <name>
     Role: <role>
+    Status: <status>
     Device Type: <manufacturer> <model> (<u_height>)
     Asset tag: <asset_tag> (if defined)
     Serial: <serial> (if defined)
     Description: <description> (if defined)
     """
     description = f'Name: {device.name}'
     description += f'\nRole: {device.role}'
+    description += f'\nStatus: {device.status.capitalize()}'
```

Comment:
Please be sure to use the [`get_FOO_display()` method](https://docs.djangoproject.com/en/5.1/ref/models/instances/#django.db.models.Model.get_FOO_display) for choice fields.
```suggestion
    description += f'\nStatus: {device.get_status_display()}'
```